### PR TITLE
Some german translations.

### PR DESCRIPTION
**Pull request by Rames507**
Additional Notes:
- "Kalt" -> "Kälte": ("cold" -> "Cold") I did this to stay in line with "Hitze" meaning "Heat" and not "hot".
- "Toxin" -> "Gift": "Toxin" is also very understandable and would probably not stand out to a native speaker. However "Gift" is 'less' of an anglicism, and it stays more consistent with the riven translations. ("Giftschaden")

### CODE REVIEW

**Comment by LuxGrey** [comment]
Occurences of 'derzeitig' could be replaced by 'aktuell'; both are perfectly valid, but 'aktuell' seems more natural and simple to me.
Concrete changes that this would entail:
```json
"app.settings.account.email.change.password": "Aktuelles Passwort",
"app.settings.account.email.change.passwordPlaceholder": "Dein aktuelles Passwort",
```

Other than that, the German translations proposed in this PR look fine to me.

**Comment by lonnstyle** [comment]
@Rames507 please make a change according to that
also, you may join our [discord server](https://discord.gg/M7BHnPS) so we can communicate easier

> Occurences of 'derzeitig' could be replaced by 'aktuell'; both are perfectly valid, but 'aktuell' seems more natural and simple to me. Concrete changes that this would entail:
> 
> ```json
> "app.settings.account.email.change.password": "Aktuelles Passwort",
> "app.settings.account.email.change.passwordPlaceholder": "Dein aktuelles Passwort",
> ```
> 
> Other than that, the German translations proposed in this PR look fine to me.

**Comment by Rames507** [comment]
Done. You're right "aktuell" does feel more natural. I guess I was a little bit tired when translating that. 

> also, you may join our [discord server](https://discord.gg/M7BHnPS) so we can communicate easier

I'm already a member of the discord (Rames507#4369), feel free to message me there if there are any issues.

File: locales/de.json [hunk 1]
```diff
@@ -247,25 +247,25 @@
   "app.auctions.search.option.sort.priceAsc": "Preis (aufsteigend)",
   "app.auctions.search.option.sort.priceDesc": "Preis (absteigend)",
   "app.auctions.search.nemesis.label.quirk": "Marotte",
-  "app.auctions.search.nemesis.placeholder.quirk": "Any",
+  "app.auctions.search.nemesis.placeholder.quirk": "Beliebig",
```
Comment (Rames507) [inline locales/de.json:250]: The translation of "Any" was hard. I chose "Beliebig". Some alternatives: "keine Einschränkungen", "Egal", "Alle"
If you have a better suggestion, go ahead.

File: locales/de.json [hunk 2]
```diff
@@ -247,25 +247,25 @@
...
   "app.auctions.search.nemesis.label.damage": "Beschädigung",
   "app.auctions.search.nemesis.label.ephemera": "Ephemera",
-  "app.auctions.search.nemesis.placeholder.ephemera.any": "Any",
+  "app.auctions.search.nemesis.placeholder.ephemera.any": "Beliebig",
   "app.auctions.search.nemesis.placeholder.ephemera.without": "Ohne",
   "app.auctions.search.nemesis.label.havingEphemera": "Mit Ephemera",
-  "app.auctions.search.nemesis.label.weaponName": "Weapon Name",
-  "app.auctions.search.nemesis.placeholder.weaponName": "Any",
+  "app.auctions.search.nemesis.label.weaponName": "Name der Waffe",
+  "app.auctions.search.nemesis.placeholder.weaponName": "Auswählen",
```
Comment (Rames507) [inline locales/de.json:258]: This is the only "Any" I decided to translate differently. I feel like "Auswählen" feels natural in a mandatory dropdown menu.

**Review by lonnstyle — APPROVED** [review]
LGTM

**Pull request merged**